src/components/HomeContent.tsx

```diff
@@ -0,0 +1,213 @@
+"use client"; // Mark this component as a Client Component
+
+import React, { useState, useEffect } from "react";
+import { Box, Container, Typography } from "@mui/material";
+import Card from "../components/Card";
+import ExploreMore from "../components/ExploreMore";
+import Feedback from "../components/Feedback";
+import { type User } from "@supabase/supabase-js";
+import supabaseClient from "~/api/supabaseConfig";
+
+interface HomeContentProps {
+	user: User | null;
+}
+
+// FeedbackData interface
+interface FeedbackData {
+	id: number;
+	name: string;
+	timeAgo: string;
```

Comment:
OKay Nice, I also think it is something that we can discuss as a group because usually apps post how long ago a post was made and not the actual time. This just reduces the complexity of managing different time zones